PWGGA/CaloTrackCorrelations/macros/QA/AddTaskCalorimeterQA.C

```diff
@@ -152,13 +151,16 @@ AliAnalysisTaskCaloTrackCorrelation *AddTaskCalorimeterQA(const char *suffix="de
   AliEMCALRecoUtils* recou = cu->GetEMCALRecoUtils();
 
   gROOT->LoadMacro("$ALICE_PHYSICS/PWGPP/EMCAL/macros/ConfigureEMCALRecoUtils.C");
-  ConfigureEMCALRecoUtils(recou,
-                          simulation,
-                          bExotic,
-                          bNonLin,
-                          bEnCalib,
-                          bBadMap,
-                          bTiCalib);  
+
+  TString recouStr = Form("(reinterpret_cast<AliEMCALRecoUtils*>(%p))", recou);
```

Comment:
Ahhh it's precisely `%p` what I was referring to, not `%x` 👍

Reply:
Yes, much simpler than using strstream...